cli/lib/tasks/download.js

```diff
@@ -346,7 +346,7 @@ const start = (opts) => {
     return getCA()
   })
   .then((ca) => {
-    return downloadFromUrl({ url, downloadDestination, progress, ca, version,
+    return downloadFromUrl({ versionUrl, downloadDestination, progress, ca, version,
```

Comment:
```suggestion
    return downloadFromUrl({ url: versionUrl, downloadDestination, progress, ca, version,
```
@JJ This should fix the unit test failures